tests/test_builtin_casters.py

```diff
@@ -323,3 +323,12 @@ def test_numpy_bool():
     assert convert(np.bool_(False)) is False
     assert noconvert(np.bool_(True)) is True
     assert noconvert(np.bool_(False)) is False
+
+
+def test_int_long():
+    """In Python 2, a C++ int should return a Python int if possible, not long."""
```

Comment:
Perhaps elaborate a bit on why:

> In Python 2, a C++ int should return a Python int rather than long if possible:  int's can always be used where a long is required, but longs are not always accepted where ints are used (such as the argument to sys.exit()).

Reply:
It is possible (though extremely rare) that a long could be required; you could just check the type, for example. But now you can force a long in C++ by casting to long long.

Reply:
Updated.